src/robot/libraries/String.py

```diff
@@ -571,6 +571,9 @@ def fetch_from_right(self, string, marker):
     def generate_random_string(self, length=8, chars='[LETTERS][NUMBERS]'):
         """Generates a string with a desired ``length`` from the given ``chars``.
 
+        ``length`` can be given as a number, a string representation of a number,
+        or as a range of numbers, such as ``5-10``.
```

Comment:
Need a note here or at the end of the documentation that this is new in Robot Framework 5.0.

Reply:
Perhaps should also more explicitly explain that when the value is given as a range, actual length is selected randomly from that range.

Reply:
Improved documentation.